### ISSUE TITLE
SearchResult is not stable

### ISSUE THREAD

**Issue by ShreyashPromact** [Quinny898/PersistentSearch#12]
Try with this.

1. Click on Search Box, it will show all the pre-search options as Search result.
2. Select one from the search result
3. Now open search box again with click on Search icon
4. Now each time you open search box, you will have that item added in to Search Result even if it is already there.

Please resolved this issue as soon as possible.

![device-2015-03-25-120805](https://cloud.githubusercontent.com/assets/8598335/6819482/adb79712-d2e7-11e4-98d9-ba805b4d2bac.png)

See above image, after doing that i have all item as Result 1 as SearchResult.

**Comment by Xpak** [comment]
try this:


if (search.getSearchables().toString().contains(searchTerm)){
                    }else{
                        search.addSearchable(new SearchResult(searchTerm,getResources().getDrawable(R.drawable.ic_history)));
                    }

**Comment by ShreyashPromact** [comment]
@Xpak  Thanks for your reply but where to put this?

**Comment by Xpak** [comment]
@ShreyashPromact right now I'm not in my workspace , but I think the method was onSearch

**Comment by ShreyashPromact** [comment]
@Xpak Ok. I will add it and check it. Thanks for the reply.

**Comment by anupsarode** [comment]
@ShreyashPromact - Did you find a solution to this? I tried the code provided by @Xpak  but that didn't help.

**Comment by ShreyashPromact** [comment]
@anupsarode   no, I am not getting any solution to this. I guess @Xpak is doing for that.

**Comment by Quinny898** [comment]
Was this fixed in the update?

**Comment by ajans** [comment]
No it wasn't fixed in the update, still reproducable in the sample app.

**Comment by ShreyashPromact** [comment]
@ajans @Quinny898 

I have resolved one issue for this library. It is related to voice search it is using. I will found some time and get this issue resolved as well. Thanks for keep notifying for such issue.

Sincerely,
Shreyash
